dev_docs/contributing/third_party_dependencies.mdx

```diff
@@ -67,6 +67,57 @@ on Github:
   [Permitted Open Source Licenses list](https://github.com/elastic/open-source/blob/main/elastic-product-policy.md#permitted-licenses-list).
 </DocCallOut>
 
+### Dependency evaluation
+
+1. **Snyk health check**
+    - Ensure a minimum health score is >= 80 in [Snyk Advisor](https://snyk.io/advisor).
+    - If the health score is below 80, a valid business justification must be provided.
+
+2. **Vulnerability assessment**. Check vulnerability reports from Snyk for the exact dependency version, available through [Snyk Advisor](https://snyk.io/advisor).
+    If vulnerabilities are present (production dependencies only):
+
+    - **Critical (CVSS 9.0 - 10.0)**:
+      - **False positive:** Proceed to Step 3.
+      - **Confirmed vulnerability:** Reject dependency.
+
+    - **High (CVSS 7.0 - 8.9)**:
+      - **False positive:** Proceed to Step 3.
+      - **Confirmed vulnerability:** Reject dependency.
+
+    - **Medium/Low (CVSS 0.0 - 6.9)**:
+      - **False positive:** Proceed to Step 3.
+      - **Confirmed vulnerability:** Exception can be granted if the vulnerability is confirmed to be impossible to exploit.
+      
+    **Note:** dev-only dependencies are exempt from these criteria and are acceptable without this evaluation since they cannot affect us in any meaningful way.
```

Comment:
cc @elastic/kibana-security for discussion

I think we should still evaluate dev dependencies, but we may choose to be more lenient with them in this interim period.

Reply:
++ I think so too - maybe we take dev deps on a case-by-case basis. IIUC some dev dependencies _could_ still potentially affect us by compromising the build process (this is not something I have expertise in, but if true should be considered). If other have more insight or can refute, please chime in.

Reply:
Agreed, we can verbally agree to be more lenient, but I don't think we should document this "exception" since we really want **all** dependencies to be vulnerability-free eventually.